ml_pipeline_gen/models.py

```diff
@@ -161,6 +163,13 @@ def _write_template(self, env, template_path, args, dest):
         with open(dest, "w+") as f:
             f.write(body)
 
+    def _write_static(self):
+        """Copies static files to the working directory."""
+        root_dir = pathlib.Path(__file__).parent.resolve()
+        for d in root_dir.joinpath("static").iterdir():
```

Comment:
This throws a `FileNotFoundError` for me. According to readme instructions, my current dir is mounted on a folder 'sklearn-demo' and it's trying to copy files from `ml-pipeline-gen/static` which doesn't exist (looked in the ml-pipeline-gen package installed from PyPI).

Reply:
oh shoot, I might need to do something with manifests